packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -95,113 +95,104 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
     [IlluminationState.NORMAL]: hintDisplayText.hintIlluminationNormalText,
   };
 
-  const getInstructionContent = () => {
-    if (isStartView) {
+  if (isStartView) {
+    return (
+      <>
+        <VisuallyHidden role="alert">
+          {hintDisplayText.hintCenterFaceInstructionText}
+        </VisuallyHidden>
+        <DefaultToast text={hintDisplayText.hintCenterFaceText} isInitial />
+      </>
+    );
+  }
+
+  if (errorState ?? (isCheckFailed || isCheckSuccessful)) {
+    return;
+  }
```

Comment:
Think we need to return `null` here? Pretty sure React below 18 doesn't gracefully handle `undefined` returned from JSX